### DISCUSSION TITLE
how to manipulate an svg stroke (or fill) colour with custom properties?

### DISCUSSION THREAD

**Discussion by yotamguttman** [webstudio-is/webstudio-community#209] [Q&A]
![image](https://github.com/user-attachments/assets/1961156e-71f9-4794-91a3-c039a7dfff07)
 this is an icon I designed and exported in .svg. when I used in on Penpot to design the website, I could simply change the stroke colour (and width) of the icon as I pleased, thanks to it being an SVG file. how can I do the same thing in Webstudio?

![image](https://github.com/user-attachments/assets/8235eeca-8889-46e8-bc29-434d1aab148d)
I presumed it'd be something in the parent's advanced properties. if I were to code this website I'd probably use stroke. so I tried that but nothing happened.

**Comment by kof** [comment]
you will have to set on svg attributes inside html embed the stroke or fill depending on what you use to "currentColor" and then set color property on html embed or any parent instance to the desired value

**Reply by yotamguttman** [reply]
worked like a charm! thank you!

there's one downside, which my be related to the way svg works rather than webstudio, but I couldn't figure out how to make a child (svg), inherit a value of the parent (stroke: red;). I had to make corresponding tokens to each svg colour variant, instead of the svg taking the colour from its parent object, its container (button), which influences the colour of both the text and the background of the button.

![image](https://github.com/user-attachments/assets/b544ae04-36d1-4dfe-bc6d-e07f15750105)
![image](https://github.com/user-attachments/assets/8b1ede45-9cce-49bd-ad09-6250269f2d0b)
